src/main/java/com/sergiofah/model/Line.java

```diff
@@ -1,12 +1,19 @@
 package com.sergiofah.model;
 
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
 import javax.persistence.*;
-import java.io.Serializable;
 import java.util.List;
 
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
 @Entity
 @Table(name = "line")
-public class Line implements Serializable {
+public class Line{
```

Comment:
The left brace must always be separated from the previous character by most Java code conventions and our convention.